snuba/migrations/runner.py

```diff
@@ -102,25 +98,20 @@ def run_migration(
         """
         migration_group, migration_id = migration_key
 
-        pending_migrations = self._get_pending_migrations(migration_key.group)
+        group_migrations = get_group_loader(migration_group).get_migrations()
 
-        if migration_key.migration_id not in [
-            m.migration_id for m in pending_migrations
-        ]:
-            # Ensure migration exists
-            try:
-                get_group_loader(migration_group).load_migration(migration_id)
-            except MigrationDoesNotExist as e:
-                raise MigrationError(e)
+        if migration_id not in group_migrations:
+            raise MigrationError("Invalid migration")
```

Comment:
Could you please add some details on why the migration is innvalid (missing in the group), since this exception is bubbled up to the CLI scripts.